src/main/java/it/gov/pagopa/cgn/portal/repository/NotificationRepository.java

```diff
@@ -4,6 +4,6 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface NotificationRepository
-        extends JpaRepository<NotificationEntity, Long> {
+        extends JpaRepository<NotificationEntity, String> {
```

Comment:
why this?

Reply:
It's a fix because NotificationEntity.Key annotated as Id is String not Long.  The type of ID in generic repository must be the same.